ArduPlane/config.h

```diff
@@ -302,8 +302,13 @@
 #endif
 
 #ifndef SOARING_ENABLED
- #define SOARING_ENABLED !HAL_MINIMIZE_FEATURES
+    #ifdef SOARING_WANTED
+        #define SOARING_ENABLED ENABLED
+    #else
+        #define SOARING_ENABLED !HAL_MINIMIZE_FEATURES
+    #endif
```

Comment:
I think you could avoid adding the SOAR_WANTED define by just adding "define SOARING_ENABLED 1" to your new hwdef - then it is already defined by this point and won't be changed.

Reply:
https://github.com/ArduPilot/ardupilot/pull/14663#discussion_r444125071
My aim of SOARING_WANTED is to avoid SOARING_ENABLED in vehicles other than Plane when using SITL.
If there is a better solution, your hint is very welcome.